src/components/ArticleById/ArticleById.jsx

```diff
@@ -1,10 +1,27 @@
 import {useParams} from 'react-router-dom'
 import { useState, useEffect } from 'react';
-import { getArticleById } from '../../api';
+import { getArticleById, patchVotesByArticleId } from '../../api';
 import './ArticleById.css'
 import CommentsById from '../CommentsById/CommentsById';
 
 function ArticleById(){
+
+    function handleVote(e, numOfVotes){
+        const voteObj = { in_votes: numOfVotes };
+        const votesElement = document.getElementById("votes");
+        const votesStr = votesElement.innerHTML;
+        let votesNum = +votesStr.substring(votesStr.indexOf(" ")+1);
```

Comment:
we shouldn't be using any getElementById or 'javascripty' ways to be setting items as we are in react.
The votes should be set in state and that way we can update the value of the votes when people interact with it.